ios/Exponent/Versioned/Core/Api/Components/SharedElement/RNSharedElementContent.m

```diff
@@ -20,20 +20,21 @@ - (instancetype)initWithData:(id) data type:(RNSharedElementContentType)type ins
 
 + (BOOL) isKindOfImageView:(UIView*) view
 {
+  NSString* className = NSStringFromClass(view.class);
   return (
           [view isKindOfClass:[UIImageView class]] ||
-          [NSStringFromClass(view.class) isEqualToString:@"RCTImageView"]
+          [className isEqualToString:@"RCTImageView"] ||
+          [className isEqualToString:@"ExpoImage.ImageView"]
           );
 }
 
 + (UIImageView*) imageViewFromView:(UIView*) view
 {
   if ([view isKindOfClass:[UIImageView class]]) {
     return (UIImageView*) view;
-  } else if ([NSStringFromClass(view.class) isEqualToString:@"RCTImageView"]) {
-    // As of react-native 0.60, RCTImageView is no longer inherited from
-    // UIImageView, but has a UIImageView as child. That will cause this code-path
-    // to be executed, where the first child view is returned.
+  } else if ([RNSharedElementContent isKindOfImageView:view]) {
+    // Both react-native and expo-image have a UIImageView
+    // as the first child.
```

Comment:
There is a chance this will not work correctly with Fabric, but since `react-native-shared-element` doesn't support it yet it's not a big problem. Let me know if you'll need any help with that in the future 😉

Reply:
Oh yeah good one, I should definitely check that one of these days 👍